server/src/test/java/com/netcracker/MessageTest.java

```diff
@@ -0,0 +1,31 @@
+package com.netcracker;
+
+import com.netcracker.domain.Message;
+import org.junit.Test;
+
+import static com.netcracker.domain.MessageType.NAME_REQUEST;
+import static com.netcracker.domain.MessageType.TEXT;
+import static org.junit.Assert.assertEquals;
+
+public class MessageTest {
+    @Test
+    public void shouldReturnMessageTypeWhenTryingToGet() {
+        Message message = new Message(TEXT, "Message text");
+
+        assertEquals(message.getMessageType(), TEXT);
+    }
+
+    @Test
+    public void shouldReturnTextWhenTryingToGet() {
+        Message message = new Message(TEXT, "Message text");
+
+        assertEquals(message.getData(), "Message text");
```

Comment:
&#x1F34E; **Issue**
Also, you specified expected and actual results in the wrong order. Upon a test failure Junit will tell you that you expected whatever the contents of "message.getData()" are, but you've got "Message text".